### ISSUE TITLE
Bad require path to SymfonyRequirements.php file

### ISSUE THREAD

**Issue by quentinus95** [sensiolabs/SensioDistributionBundle#324]
I'm not sure whereas this issue belongs to this repository.

I'm experiencing a new issue with multiple projects; the path to the `SymfonyRequirements.php` file is badly generated during composer install.

Here is the error trace:

```
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::prepareDeploymentTarget
PHP Warning:  require_once(/home/quentin/Git/web/project/bin/./SymfonyRequirements.php): failed to open stream: No such file or directory in /home/quentin/Git/web/project/bin/symfony_requirements on line 4
PHP Fatal error:  require_once(): Failed opening required '/home/quentin/Git/web/project/bin/./SymfonyRequirements.php' (include_path='.:') in /home/quentin/Git/web/project/bin/symfony_requirements on line 4
```

The bin file `./bin/symfony_requirements` is actually wrong:

```php
#!/usr/bin/env php
<?php

require_once dirname(__FILE__).'/./SymfonyRequirements.php';
```

I'm working with `sensio/distribution-bundle ^5.0 [5.0.20]` and `symfony/symfony ^3.3 [3.3.6]`.

(I also have this issue with API-Platform.)

**Comment by Seldaek** [comment]
See https://github.com/composer/composer/issues/6599#issuecomment-322332559 for possible explanation/solution, if someone wants to send a PR.

**Comment by fabpot** [comment]
fixed in bf582cc
